sigs/cncf-sigs.md

```diff
@@ -111,8 +111,23 @@ As a starting point let’s be inspired by CNCF OSS Projects and by K8s SIGs.  T
 
 #### Tech Lead
 
-* Leads projects in the SIG’s area.
-* Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
+A Tech Lead will typically have a track record of participation in SIG
+activities and/or as TOC Contributors; however, this is not a requirement,
+particularly for new SIGs and to fill vacancies.
+
+Qualifications:
+* Expert in the SIG area
```

Comment:
I think you omitted "leads projects in the SIG's area"?  Was this intentional?

Reply:
I think there was a conversation about this somewhere else - I don’t think you have to be a project lead to be a SIG tech lead - we will have a bigger pool of experts to draw on that way